File: PCMC/Entities/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace PCMC.Entities
{
    public class Student
    {
        [Key]
        public int ID { get; set; }
        public User User { get; set; }
        public School SchoolEnrolled { get; set; }
        public Team TeamAssigned { get; set; }
    }
}

File: PointSaaS/Client/Common/POCO/Student.ts
class Student {
    public constructor(id: number, usr: User, school: School, team: Team) {
        this.ID = id;
        this.User = usr;
        this.SchoolEnrolled = school;
        this.TeamAssigned = team;
    }

    public ID: number;
    public User: User;
    public SchoolEnrolled: School;
    public TeamAssigned: Team;
}